e2e/player-init.spec.ts

```
@@ -0,0 +1,20 @@
+import { test, expect } from "./fixtures/player.fixture";
+
+test.describe("Player render & initialization", () => {
+  test("1-1: player is rendered in the DOM", async ({ playerPage }) => {
+    await expect(playerPage.player).toBeVisible();
+  });
+
+  test("1-2: play button is visible on initial load", async ({ playerPage }) => {
```

ESLint

[error] Replace `·playerPage` with `⏎····playerPage,⏎·`

(prettier/prettier)